tensorflow/compiler/mlir/tosa/transforms/legalize_tfl.cc

```diff
@@ -1618,6 +1619,23 @@ LogicalResult ConvertTFLTransposeConvOp::matchAndRewrite(
   bool output_is_qtype =
       output_type.getElementType().isa<mlir::quant::QuantizedType>();
 
+  const bool has_bias = !(!tfl_conv_op.getBias() ||
+                          tfl_conv_op.getBias().getType().isa<NoneType>());
+
+  if (has_bias) {
+    RankedTensorType bias_type =
+        dyn_cast<RankedTensorType>(tfl_conv_op.getBias().getType());
+    bool bias_is_qtype =
+        bias_type.getElementType().isa<mlir::quant::QuantizedType>();
+
+    if (input_is_qtype != bias_is_qtype) {
+      return rewriter.notifyMatchFailure(
+          op,
+          "input/bias tensor should "
```

Comment:
Formatting here seems weird.

Reply:
I have left this as is for now, as there are instances of this function being used with this formatting across the file. But do let know if this needs looking at again before it can be merged!